src/shared/ui/combobox/types.ts

```diff
@@ -23,5 +23,5 @@ export interface ComboboxProps<T> extends SelectAdapter<T> {
   searchDelay?: number;
   searchPlaceholder?: string;
   selectPlaceholder?: string;
-  value?: T;
+  initialValues?: T[];
```

Comment:
Needs to be initialValuesIds?: string[] after resolving https://alcosi.atlassian.net/browse/TWINFACES-207